examples/docs_snippets/docs_snippets_tests/concepts_tests/types_tests/test_types.py

```diff
@@ -1,7 +1,7 @@
 import pytest
+from docs_snippets.concepts.types.types import test_dagster_type
```

Comment:
Do we know why it is ordering dagster after docs_snippets

Reply:
Yes-- it is because `dagster` is right now recognized as the only "first-party" package. There is an extended discussion of this here with charlie here: https://github.com/charliermarsh/ruff/issues/1069#issuecomment-1339250929

Charlie is working on a solution that will get the current package treated as first-party. Until that lands (and some other relevant config tweaks) I decided to no try to force it with config, since previously isort was doing the same thing for all our other packages (if you look at e.g. `dagster-graphql` you'll see that `dagster` imports go below `dagster_graphql` imports).